README.md

```diff
@@ -1,2 +1,39 @@
 # signoff-pxb
 Repos for pxb related release sign-off
+
+Each PX-Backup release need to follow this sign-off process and share all artifacts before releasing to production.
+PX team need to share the compatibility metrics as well for release. 
+
+
+## Following are the tests scope for PX-Backup
+
+- Installation tests
+- Torpedo tests
+- Functional tests
+- Market place tests
+- Integration tests
```

Comment:
Integration and Functional Tests are same here.

Reply:
both are different, integration tests for vpc block csi driver and pxe. may be Torpedo covering all